resources/css/style.css

```diff
@@ -590,4 +602,67 @@ video {
         max-width: 1170px;
         /* margin: 5rem auto; */
     }
+
+}
+
+@media only screen and (max-width: 414px){
+
+    .footer-buttons {
+      width: 100%; //override the web width attribute
+    }
+
+    .float-right, .col-block{
+      float:none;
+    }
+
+    .fcontainer {
+      padding: 0%;
+    }
+
+    .footer-buttons .footer-buttons-wrapper {
+      margin: 0 40px 0 40px;
+    }
+
+    .footer-buttons .button {
+      box-sizing: content-box;
+      display: block;
+      float: none;
+      width: auto;
+    }
+
+    .footer-logo, .footer-aboutus-section, .footer-oursolutions-section, .footer-ourwork-section, .footer-moreinfo-section,
+    .address {
+      display: none;
+    }
+
+    .footer-sections {
+      margin-left: 0;//reset web margin
+      width: 100%;
+    }
+
+    .footer-sections address {
+      display: block;
+      text-align: center;
+      width: 100%;
+      padding-top: 2em;
+    }
+
+    .section-header, .footer-list-item-anchor, .social, .footer-copyright{
+      display: block;
+    }
+
+    .footer-list-item-anchor li {
+      text-align: center;
+      padding: 5px;
+    }
+
+    .footer-sections .social {
+      padding-bottom: 50px;
+    }
+
+    .footer-copyright {
+      min-height: 300px;
+      margin-top: 0;
+      margin-bottom: 50px;
```

Comment:
There's a lot of extra space, both black background space and whitespace at the bottom of the mobile view. I tested this in both Chrome and Firefox.

   ![Screenshot](https://framapic.org/i6hZtJc2DRJi/eZIKwKBS5upW)

   The extra space seems to be related to this:

   ```
   .footer-copyright {
       min-height: 300px;
       margin-top: 0;
       margin-bottom: 50px;
   }
   ```

   Removing that bottom margin removes the whitespace. The extra black background space is from the `min-height: 300px`. Removing both of them and adding padding might solve the problem:

   ```
   .footer-copyright {
       margin-top: 0;
       padding: 20px;
   }
   ```